src/uwp/Microsoft.Build.Net.CoreRuntimeTask/ResourceHandlingTask.cs

```diff
@@ -267,18 +323,82 @@ private void WriteStateFile(string stateFile, ResourceHandlingState state)
             }
         }
 
-        private string ExtractFrameworkAssemblyResW(string assemblyFilePath, out bool containsResources)
+        private ResWInfo ExtractResourcesFromStream(Stream stream, IAssembly assembly, string resourceFileName, bool containsFrameworkResources)
         {
-            string assemblyName;
-            using (Stream stream = ExtractFromAssembly(assemblyFilePath, out assemblyName, out containsResources))
+            string reswFilePath;
+            string resourceIndexName;
+            string neutralResourceLanguage = "";
+
+            if (containsFrameworkResources)
             {
-                if (stream == null)
-                    return null;
+                reswFilePath = OutResWPath + Path.AltDirectorySeparatorChar + resourceFileName + ".resw";
+                resourceIndexName = resourceFileName;
+                neutralResourceLanguage = "en-US";
+            }
+            else {
```

Comment:
>else { [](start = 12, length = 6)

nit: please move the parenthesis to new line

Reply:
Ok.